en/sensor/pmw3901.md

```diff
@@ -0,0 +1,76 @@
+# PMW3901 Flow Sensor
+
+PMW3901 is an [optical flow](../sensor/optical_flow.md) asic that computes the flow internally and provides a difference in pixels between 
+each frame over a SPI interface. It is basically a tracking sensor similar to what you would find in a computer mouse, but adapted to work between 
```

Comment:
People do search on component part numbers, but they just as much (or more) look for products. So it is worth highlighting the relevant products as early as possible.

How about after the sentence ending in "SPI interface" have another one listing the manufacturer parts  - "Bitcraze" then a paragraph break and following sentence.